app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/protectiveword/ProtectiveWordViewModel.kt

```diff
@@ -41,20 +41,20 @@ class ProtectiveWordViewModel @Inject constructor(
             val isRecordsAvailable = medicationRecordRepository.isMedicationRecordsAvailableForPatient(patientId)
             if (isRecordsAvailable) {
                 if (isProtectiveWordValid(protectiveWord)) {
-                    _uiState.tryEmit(
+                    _uiState.emit(
                         FetchMedicationUiState(
                             isRecordsUpdated = true
                         )
                     )
                 } else {
-                    _uiState.tryEmit(
+                    _uiState.emit(
                         FetchMedicationUiState(
                             wrongProtectiveWord = true
                         )
                     )
                 }
             } else {
-                _uiState.tryEmit(
+                _uiState.emit(
```

Comment:
use update instead of emit
also, take reference from IndiVidual Health record view model